pallets/moonbeam-orbiters/src/lib.rs

```diff
@@ -307,7 +319,13 @@ pub mod pallet {
 					&T::OrbiterReserveIdentifier::get(),
 					&orbiter,
 					min_orbiter_deposit,
-				)
+				)?;
+				RegisteredOrbiter::<T>::insert(&orbiter, true);
```

Comment:
Changes benchmark, although if this is a rush I dont think there is much issue with this